seqr/utils/logging_utils.py

```diff
@@ -73,14 +73,16 @@ def log_model_update(logger, model, user, update_type, update_fields=None):
     logger.info('{} {} {}'.format(update_type, db_entity, entity_id), user, db_update=db_update)
 
 
-def log_model_bulk_update(logger, models, user, update_type, update_fields=None):
+def log_model_bulk_update(logger, models, user, update_type, update_fields=None, parent=None):
     if not models:
         return []
     db_entity = type(models[0]).__name__
-    entity_ids = [o.guid for o in models]
+    entity_ids = [o.guid if hasattr(o, 'guid') else o.id for o in models]
```

Comment:
this is not the logging we care about here. for models without guids, write a different helper function, there is no need to log a list of database IDs just log the number of models